continuous_integration/recipe/meta.yaml

```diff
@@ -43,6 +43,8 @@ requirements:
     - prompt-toolkit >=3.0.8
     - pygments >=2.7.1
     - tabulate
+  run_constrained:
+    - __glibc >=2.32  # [aarch64]
```

Comment:
For the ARM wheels, think we need to do something like

```
wheel tags --platform-tag=manylinux_2_32_aarch64 <filename>.whl
```

Reply:
This is being handled in PR ( https://github.com/dask-contrib/dask-sql/pull/1152 ) with commit ( https://github.com/dask-contrib/dask-sql/pull/1152/commits/3284141fc83f50dbfbd38236f448f1e33f09043c )